app/views/complaints/_tta_create_activity_report.html.erb

```diff
@@ -1,15 +1,24 @@
 <div class="ct-section-divider">
```

Comment:
[question] do we need a similar thing for the `_tta_edit_activity_report` partial? (also did i goof -- should that be `update` and not `edit`?)

Reply:
So... I think we need to break this off into another story. Right now errors on edit will open this form instead of but I think fixing that will require a not insignificant number of changes.

As for the naming, if anything I think this one should be `_tta_new_activity_report.html.erb` rather than changing the edit one, but both forms work for me

Reply:
works for me!